docs/source/index.md

```diff
@@ -0,0 +1,109 @@
+---
+sd_hide_title: true
+---
+
+# Scylla Operator Documentation
+
+:::{toctree}
+:hidden:
+:maxdepth: 1
+
+architecture/index
+installation/index
+resources/index
+quickstarts/index
+support/index
+api-reference/index
+:::
+
+## Scylla Operator
+
+::::{grid} 1 1 1 2
+:reverse:
+
+:::{grid-item}
+:columns: 12 12 12 4
+
+```{image} logo.png
+:width: 150pt
+:class: sd-m-auto
+:name: landing-page-logo
+```
+
+:::
+
+:::{grid-item}
+:child-align: justify
+:columns: 12 12 12 8
+:class: sd-d-flex-column
+
+{{productName}} project helps users to run ScyllaDB on Kubernetes. 
+It extends the Kubernetes APIs using [CustomResourceDefinitions(CRDs)](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) and runs controllers that reconcile the desired state declared  using these APIs.
+
+{{productName}} works with both ScyllaDB Open Source and ScyllaDB Enterprise. You only have to [change the ScyllaCluster image repository](#scyllacluster-enterprise) and [adjust the ScyllaDB utils image using ScyllaOperatorConfig](resources/scyllaoperatorconfigs.md#tuning-with-scylladb-enterprise)
```

Comment:
> By default, our examples use ScyllaDB Open Source

this could actually be changed - I was thinking about a ScyllaOperatorConfig value that changes the default repo
 but I guess we can update the docs if it comes to that